contrib/cirrus/runner.sh

```diff
@@ -102,7 +102,7 @@ function _run_bindings() {
 function _run_docker-py() {
     m="Testing of API was performed using the **DOCKER** client"
     warn "$m"
-    source venv/bin/activate
+    source .venv/docker-py/bin/activate
```

Comment:
Gonna need the `venv -> .venv` change in `_run_apiv2()` also.

Reply:
In my version of _run_apiv2(), there is no `venv`

Reply:
Correct, you added it in `setup_environment.sh`.  The `runner.sh` script executes in a different shell session, so if you want what's inside, you'll need to `source .venv/docker-py/bin/activate` again.  Does that make sense?

Reply:
...what's inside ***the venv***, you'll need to...